docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN --mount=type=cache,target=/var/cache/apt \
     apt-get install -qqy \
     git build-essential make curl gpg wget sudo nano file procps bash
 
-RUN curl -L https://go.dev/dl/go${GO_VERSION}.linux-arm64.tar.gz | tar -xzv -C /usr/local && \
+RUN curl -L https://go.dev/dl/go${GO_VERSION}.linux-$(if [ `dpkg --print-architecture` = arm64 ]; then echo arm64; else echo armv6l; fi).tar.gz | tar -xzv -C /usr/local && \
```

Comment:
This should work, so just confirm once it's built that go is indeed arm arch (when building that version.)